src/wasm-lib/kcl/src/parsing/parser.rs

```diff
@@ -2280,23 +2280,33 @@ struct ParamDescription {
     labeled: bool,
     arg_name: Token,
     type_: std::option::Option<FnArgType>,
-    is_optional: bool,
+    default_value: Option<DefaultParamVal>,
 }
 
 fn parameter(i: &mut TokenSlice) -> PResult<ParamDescription> {
-    let (found_at_sign, arg_name, optional, _, type_) = (
+    let (found_at_sign, arg_name, question_mark, type_, _ws, default_literal) = (
         opt(at_sign),
         any.verify(|token: &Token| !matches!(token.token_type, TokenType::Brace) || token.value != ")"),
         opt(question_mark),
-        opt(whitespace),
```

Comment:
Up to you if you think this is right. But I'd bet that someone will report a bug to add this line back.

Reply:
Oops, I didn't realize I had removed this, thanks.